$OpenBSD: patch-src_PrefsUnifiedDlg_h,v 1.1 2016/10/10 13:25:38 dcoppa Exp $

commit 8f0c06698f988e8d93b62045716bd205a015490b
Author: upstream svn <svn@amule.org>
Date:   Thu Oct 6 10:49:44 2016 +0000

Fix restoring toolbar orientation on 'Cancel'

Since the toolbar orientation change takes effect immediately, we have to be
extra careful to restore it to its original state if the user presses 'Cancel'.
Otherwise the current GUI layout and the preferences would get out of sync.

--- src/PrefsUnifiedDlg.h.orig	Fri Sep 16 09:55:06 2016
+++ src/PrefsUnifiedDlg.h	Mon Oct 10 15:02:43 2016
@@ -128,6 +128,10 @@ class PrefsUnifiedDlg : public wxDialog (protected)
 	void OnInitDialog( wxInitDialogEvent& evt );
 
 	DECLARE_EVENT_TABLE()
+
+private:
+	bool	m_verticalToolbar;
+	bool	m_toolbarOrientationChanged;
 };
 
 #endif
